src/helpers/index.js

```diff
@@ -1,4 +1,14 @@
-const isSevere = (mans, vehicles) => mans > 20 || vehicles > 5;
+const moment = require('moment');
+
+const OCCURENCE_DATE_FORMAT = 'DD/MM HH:mm';
+
+const isSevere = (time, mans) => {
+  const formattedBegin = moment(time, OCCURENCE_DATE_FORMAT).subtract(1, 'hours');
```

Comment:
When occurrence hour bug (occurrence time is one hour ahead) is fixed , the code `.subtract(1, 'hours')` should be removed.